editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Question.tsx

```diff
@@ -57,13 +57,18 @@ const Question: React.FC<Props> = React.memo((props) => {
     <>
       <Hanger hidden={isDragging} before={props.id} parent={parent} />
       <li
-        className={classNames("card", "decision", {
-          isDragging,
-          isClone: isClone(props.id),
-          isNote: childNodes.length === 0,
-          wasVisited: props.wasVisited,
-          hasFailed: props.hasFailed,
-        })}
+        className={classNames(
+          "card",
+          "decision",
+          "type-" + TYPES[props.type as TYPES],
```

Comment:
Ha! I forgot about this neat little trick.

I _think_ this means we need to update the "how to add a component" guide to say we need to add a style for a new component also (not sure how this would handle defaults / new components right now).

Reply:
In this case the style is only being used/applied as an enhancement for `type-Section`, so we wouldn't need a style unless it needed to be distinctive from the other components. As long as the type is set on a new component it'll pull through and be used if necessary (if I understand your point correctly).